mito-ai/style/ChatTaskpane.css

```diff
@@ -143,10 +143,59 @@
     align-items: center;
 }
 
-.message-start-editing-button {
+.message-action-buttons {
     position: absolute !important;
     bottom: 8px !important;
     right: 8px !important;
+    display: flex;
+}
+
+.message-action-buttons button {
+    cursor: pointer;
+    opacity: 0;
+    transition: opacity 0.2s ease-in-out, background-color 0.2s ease-in-out;
+    background-color: var(--jp-layout-color3);
+    border: none;
+    padding: 0px;
+    border-radius: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: var(--grey-900) !important;
+    width: 28px;
+    height: 28px;
+    position: relative;
+}
+
+.message-action-buttons button svg {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
```

Comment:
Do we need to be doing this at the svg level? Isn't it enough to just place the message-action-buttons in the right location which we are already doing?